### PR TITLE
Add dev gems to Gemfile, drop Gemfile.lock from VCS

### PR DESCRIPTION
**Pull request by kke**
On Ruby 2.6 `bundle exec irb` fails unless you have irb in the bundle.

The gemfile.lock in the repo seems to cause more problems than benefits.

Added byebug while at it.

### CODE REVIEW

**Comment by kke** [comment]
Rubocop gets updated because of this and causes a lot of failures. Fixing.

**Comment by jakolehm** [comment]
Conflicts...

**Comment by kke** [comment]
Confict be gone

**Comment by jakolehm** [comment]
Conflict again..

**Comment by kke** [comment]
Conflict away again

File: pharos-cluster.gemspec [hunk 1]
```diff
@@ -49,4 +49,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rspec", "~> 3.7"
   spec.add_development_dependency "fakefs", "~> 0.13"
   spec.add_development_dependency "rubocop", "~> 0.57"
+  spec.add_development_dependency "irb", "~> 1.0"
```
Comment (jakolehm) [inline pharos-cluster.gemspec]: Should we switch to `pry`?

Comment (kke) [reply pharos-cluster.gemspec]: We can try

**Comment by kke** [comment]
Back in sync again.

**Pull request merged**